meta/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+galaxy_info:
+  role_name: opencast_mariadb
+  namespace: elan
+  author: Timo Nogueira Brockmeyer
+  description: Install MariaDB for Opencast.
+  company: virtUOS
+  license: BSD-3-Clause
+  min_ansible_version: 2.09
+  platforms:
+    - name: EL
+      version:
+        - 8
```

Comment:
Should we add EL9 by now as well?

Reply:
There don't seem to be any official CentOS 9 or CentOS Stream 8 images available yet. So i will postpone changing this for now.

Reply:
After being able to test for stream 8, stream 9 would have been nice, but I can currently not get it to work on 9. So i will postpone this.